src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -15,9 +15,10 @@ import { ICON_TYPES } from "../calcite-pick-list/resources";
 import { getSlotted } from "../../utils/dom";
 
 /**
- * @slot secondary-action - A slot intended for adding a `calcite-action` or `calcite-button` to the right side of the card.
- * This is placed at the end of the item.
- */
+  * @slot secondary-action - **[DEPRECATED]** use actions-end instead.
```

Comment:
Is this really a deprecation?

Reply:
It would be nice if it were to go away in the next release or two. :)

Reply:
Can we just remove them now and note them in the migration steps?

Reply:
I'll do it!